src/pkg/backup/backup.go

```diff
@@ -89,6 +88,8 @@ func New(
 		}
 	}
 
+	fmt.Printf("\n-----\nskipped %+v %v %v\n-----\n", malware, notFound, otherSkips)
+
```

Comment:
guessing this should be removed as it's a debug statement?
```suggestion
```

Reply:
ayup!